packages/react-text/src/typographyStyles/typographyStyles.ts

```diff
@@ -0,0 +1,72 @@
+import { MakeStylesStyleFunctionRule } from '@fluentui/make-styles';
+import { Theme } from '@fluentui/react-theme';
+
+/**
+ * Make-styles rules for the typography variants
+ */
+export const typographyStyles: {
```

Comment:
having an object like this makes the styles un-treeshakeable, so using a `title1` style results in all typography stles in the bundle.

It might not be a big deal though

Reply:
It also does seem like a nicer usage pattern to have separate exports:

```js
import { bodyStyles } from 'react-text'
```

WDYT ?

Reply:
Good point, it makes sense to have separate imports. However, I think the DX might suffer a bit here as something like `title1Styles` from `react-text`, together with the Text component, hooks, etc in the same package, might not be immediately clear to the user about what these are.
Can you think of any way to provide this context to the user or group them? Apart from having a separate package like `react-typography`, I can't think of anything 😕

Reply:
At least in the past this was done with deep imports, but let's not get into that 😨

I don't think the DX is too bad, the names of the imports should be quite readable

Reply:
Applied the suggestion with @layershifter's suggestion